lib/pbench/test/functional/conftest.py

```diff
@@ -13,13 +15,48 @@ def setup(request, pytestconfig):
 
 @pytest.helpers.register
 def capture(command):
+    """Capture the command output and return the stdout, stderr,
+    and errorcode.
+    """
     proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = proc.communicate()
     return out, err, proc.returncode
 
 
 @pytest.fixture
 def pbench_run(tmpdir):
-    pbench_run = tmpdir / "test/var/lib/pbench-agent"
+    """Create an agent run directory for a unittest."""
+    pbench_run = tmpdir / "var/lib/pbench-agent"
     os.makedirs(pbench_run)
     yield pbench_run
+
+
+@pytest.fixture
+def pbench_install_dir(tmpdir):
+    """Create an agent install directory for a unittest"""
+    pbench_install_dir = tmpdir / "opt/pbench-agent"
+    os.makedirs(pbench_install_dir)
+    yield pbench_install_dir
+
+
+@pytest.fixture
+def pbench_agent_config(tmpdir, pbench_run, pbench_install_dir, pytestconfig):
```

Comment:
We might want to consider reusing what we have, or at least including this in the refactoring: https://github.com/distributed-system-analysis/pbench/blob/master/lib/pbench/test/unit/agent/conftest.py#L8